index.js

```diff
@@ -74,10 +77,14 @@ module.exports = {
     "declaration-colon-newline-after": null,
     "declaration-empty-line-before": null,
     "function-name-case": null,
-    "max-nesting-depth": 3,
+    "max-nesting-depth": [3, {
+      ignoreAtRules: [
+        "for",
+        "include"
+      ]
+    }],
     "number-leading-zero": null,
     "selector-max-compound-selectors": null,
-    "shorthand-property-no-redundant-values": null,
-    "unit-blacklist": ["px", { "severity": "warning" }]
```

Comment:
@fabianmebus Originally, this rule was added to make the transition to `rem` easier and stop using `px` everywhere. Is it safe to remove it globally?

Reply:
@marvinhagemeister We are using this sass `rem()` function in many projects
```sass
@function rem($value) {
  @if (type-of($value) == number) {
    @if (unit($value) != 'px') {
      @error '`#{$value}` needs to be a pixel value.';
    }
  } @else {
    @error '`#{$value}` needs to be a number.';
  }

  @return $value / 16px * 1rem;
}
```
We intentionally wrote the function this way that you have to use `px` in the argument.
```sass

.foo {
  font-size: rem(42px);
}
```
I didn't find a way in that stylelint rule to ignore or exclude the `px` unit inside sass functions. Because of so many warnings caused by that rule it was almost hard to find the *real* errors.

Reply:
Fair point, thanks for clearing this one up 👍